lib/components/atoms/Video/Video.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+/**
+ *
+ * Video
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Video.style';
+import type { Props } from './types';
+
+const Video = ({
+  width,
+  height,
+  autoPlay,
+  controls,
+  loop,
```

Comment:
Naming convention for props can be better

Reply:
ok